crates/bevy_winit/src/lib.rs

```diff
@@ -41,195 +47,49 @@ pub struct WinitPlugin;
 
 impl Plugin for WinitPlugin {
     fn build(&self, app: &mut App) {
+        let event_loop = EventLoop::new();
+        app.insert_non_send_resource(event_loop);
+
         app.init_non_send_resource::<WinitWindows>()
             .init_resource::<WinitSettings>()
             .set_runner(winit_runner)
-            .add_system_to_stage(CoreStage::PostUpdate, change_window.label(ModifiesWindows));
+            // TODO: Verify that this actually works and does not cause any race-conditions or strange ordering issues
```

Comment:
I don't know how to best verify this

Reply:
This should be fine, I haven't noticed anything weird with this, so I'll just remove the comment here.